core/sis-utility/src/main/java/org/apache/sis/measure/LinearConverter.java

```diff
@@ -124,8 +124,8 @@
      * Returns a linear converter for the given scale and offset.
      */
     private static LinearConverter create(final double scale, final double offset, final double divisor) {
-        if (offset == 0) {
```

Comment:
Thanks for the proposal. Actually this condition is on two lines for similar reasons than what we had in a previous change request: today we check only for the `(scale == divisor)` condition after `(offset == 0)`, but future versions may had more lines. In particular we could use a specialized implementation in the common cases where `divisor == 1`.